File: 4271-find-the-degree-of-each-vertex/find-the-degree-of-each-vertex.py
class Solution:
    def findDegrees(self, matrix: list[list[int]]) -> list[int]:
        ans = []
        n = len(matrix)
        m = len(matrix[0])
        for i in range(n):
            count = 0
            for j in range(m):
                if matrix[i][j] == 1:
                    count += 1
            ans.append(count)
        return ans

        

File: 4271-find-the-degree-of-each-vertex/find-the-degree-of-each-vertex.cpp
class Solution {
public:
    vector<int> findDegrees(vector<vector<int>>& matrix) 
    {
        vector<int> ans;
        int n = matrix.size();
        int m = matrix[0].size();

        for(int i=0 ;i < n ; i++)
        {
            int count = 0;
            for(int j=0 ; j < m; j++)
            {
                if ( matrix[i][j] == 1)
                {
                    count++;
                }
            }
            ans.push_back(count);
        }
        return ans;
    }
};